.circleci/config.yml

```diff
@@ -105,6 +105,15 @@ jobs:
           https://github.com/go-redis/redis
           contrib/go-redis/redis/vendor/github.com/go-redis/redis
 
+    - run:
+        name: Vendor github.com/googleapis/gnostic for k8s.io/client-go
+        # This step checks out k8s.io/client-go and vendors 
```

Comment:
Should we add a TODO here? Something like:
```sh
# TODO(knusbaum): remove this once the breaking change is resolved or propagated
```
Otherwise we might end up with more inconsistencies later.

Reply:
👍